src/components/HURUmap/Panel/MobilePanel/index.js

```diff
@@ -29,8 +38,35 @@ function MobilePanel({ scrollToTopLabel, activeType, ...props }) {
     window.scrollTo({ top: 0, behavior: "smooth" });
   };
 
+  const handleClose = (e) => {
+    const code = e.nativeEvent?.target?.dataset?.value;
+    if (code && onSelectLocation) {
+      onSelectLocation({ code });
+    }
+  };
+
   return (
     <div className={classes.root}>
+      {items?.length === 0 && (
+        <Section>
+          <LocationHeader
+            variant="primary"
+            icon={Print}
+            title={geography.name}
+            {...geography}
+          />
+          <PinAndCompare
```

Comment:
Does mobile support PinAndCompare?

Reply:
@kilemensi it can, but according to design documentation, comparison should only be for desktop.